### ISSUE TITLE
OCAT 1.4 crashes Talos Principle and Doom under VK on startup

### ISSUE THREAD

**Issue by rys** [GPUOpen-Tools/OCAT#68]
AMD virtualization team reports that 1.4.216 causes startup crashes in Doom and Talos Principle under Vulkan.

**Comment by aurolou** [comment]
I was able to reproduce the bug, that Doom or also Strange Brigade Vulkan crashes on launch when OCAT's overlay is hooked in. However, I found a workaround by disabling the SteamFossilizeVulkanLayer (e.g. setting the registry of the layer to 1). In this case, the game was able to start with OCAT's enabled overlay. I am now looking into what SteamFossilizeVulkanLayer does to interfere with the OCAT layer.
